packages/gatsby-plugin-vercel-builder/README.md

```diff
@@ -0,0 +1,20 @@
+# @vercel/gatsby-plugin-vercel-builder
+
+This plugin generates [Vercel Build Output API v3](https://vercel.com/docs/build-output-api/v3) for Gatsby v4 projects.
```

Comment:
```suggestion
This plugin generates [Vercel Build Output API v3](https://vercel.com/docs/build-output-api/v3) for Gatsby v4+ projects.
```

Reply:
Not merging this until we figure out what we are actually doing for 4+